### PR TITLE
Add support for Freescale P2041RDB

### PR DESCRIPTION
**Pull request by AmyFsl**
Add support for Freescale P2041RDB

    Add platform specific machine configuration for P2041RDB platform.
      - u-boot: 2015.10-rc1
      - kernel: 4.1.12

### CODE REVIEW

File: build-config/scripts/onie-mk-bin.sh [hunk 1]
```diff
@@ -31,7 +31,7 @@ uimage_max_size=$(( 4 * 1024 * 1024 ))
 
 # Most platforms have a max u-boot size of 512KB.  You can override
 # this in the platform specific conf file.
-uboot_max_size=$(( 512 * 1024 ))
+uboot_max_size=$(( 768 * 1024 ))
```
Comment (cbrune) [inline build-config/scripts/onie-mk-bin.sh]: The value here is the default for all platforms.   This value can be overridden for a specific platform in the machine's onie-rom.conf file.

Comment (cbrune) [inline build-config/scripts/onie-mk-bin.sh]: This is a common file used by all platforms.  This value can be overridden on a per platform basis by setting uboot_max_size in machine/fsl/fsl_p2041rdb/onie-rom.conf.

**Comment by cbrune** [comment]
machine/fsl/fsl_p2041rdb/rootconf/sysroot-lib-onie/init-platform is not needed as the functions are adding no value.  This is likely a left over from copying the demonstration platform.  This file can safely be removed.

**Comment by cbrune** [comment]
I see u-boot 2015.10 is currently available.  I would base this system on the official release instead of a release candidate version:

ftp://ftp.denx.de/pub/u-boot/u-boot-2015.10.tar.bz2

**Comment by cbrune** [comment]
This file can be removed:

machine/fsl/fsl_p2041rdb/test/fsl_p2041rdb_dut.py

That is a left over from a previous testing architecture that is no longer used.

**Comment by cbrune** [comment]
All in all this looks really good. Very nice!

Just a couple of minor things to clean up.

**Comment by cbrune** [comment]
Great.  Thanks for your contribution!

@ChangeTip send @AmyFsl a processor!

Pushed to master:

- b2cdf59 u-boot: add u-boot version 2015.10
- d8f014d Add support for Freescale P2041RDB

**Comment by changetip** [comment]
Hi @amyfsl, @cbrune sent you a tip worth a processor (7,111 bits/$2.56), and I'm here to deliver it ➔ **[collect your tip](https://www.changetip.com/collect/925076)**.



**[Learn more about ChangeTip](https://www.changetip.com/tip-online/github)**

**Pull request closed without merging**